integration.go

```diff
@@ -19,6 +21,13 @@ type AWSIntegrationFragment struct {
 	OwnershipTagKeys     []string `graphql:"ownershipTagKeys"`
 }
 
+type AzureResourcesIntegrationFragment struct {
+	TenantId       string        `graphql:"tenantId"`
+	SubscriptionId string        `graphql:"subscriptionId"`
+	LastSyncedAt   *iso8601.Time `graphql:"lastSyncedAt"`
+	Aliases        []string      `graphql:"aliases"`
```

Comment:
additionally it seems like this is missing some of the fields from the input? 🤔

Reply:
Specifically
```
OwnershipTagKeys
TagsOverrideOwnership
```

Reply:
👍 These are being added